packages/xrpl/src/models/methods/baseMethod.ts

```diff
@@ -38,6 +38,7 @@ export interface BaseResponse {
   warnings?: ResponseWarning[]
   forwarded?: boolean
   api_version?: number
+  meta_blob?: string
```

Comment:
`meta_blob` needs to be at the same depth/level as `meta` field i.e. within `response.result`. At the moment, I can access both `response.result.meta_blob` and `response.meta_blob`.

Docs for tx request: https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/transaction-methods/tx/

This inclusion of `meta_blob` is not needed. 

I didn't mean to confuse you, I originally wanted a clarification on how `meta_blob` was being inherited across the classes.

Reply:
`meta_blob` is already included in `TxResponse<Transaction>.result : BaseTxResult` -- you were initially correct, there is no need for this line

Reply:
I reverted the change.